front/pages/edit.vue

```diff
@@ -533,4 +624,26 @@ export default {
 .vc-chrome-fields:last-child {
   display: none;
 }
+
+.pixelButton span {
+  mix-blend-mode: difference;
```

Comment:
I'm against using mix-blend-mode at all, for example contrast is bad for `#DB5353` pixel color.

Reply:
Instead, use full-blown black and white (`#fff`, `#000`) and switch based on lightness so contrast is at least 50% on every possible color

Reply:
As a temporary solution, I would rather set a white text and a black text-shadow (as an outline or something).